learningequality/studio
# Extract media duration on upload

## Summary
### Description of the change(s) you made

- Extracts the media duration when uploading audio or video files
- Maps the maximum duration to the related content node on publish

### Manual verification steps performed
1. Upload a video
2. Verify duration is saved in DB
3. Publish channel
4. Verify duration exists in channel database export

## References

Resolves https://github.com/learningequality/studio/issues/3078

## Comments


----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
